docs/ingestion/native-batch.md

```diff
@@ -1127,9 +1127,10 @@ Sample specs:
 |type|This should be `hdfs`.|None|yes|
 |paths|HDFS paths. Can be either a JSON array or comma-separated string of paths. Wildcards like `*` are supported in these paths. Empty files located under one of the given paths will be skipped.|None|yes|
 
-You can also ingest from cloud storage using the HDFS input source.
-However, if you want to read from AWS S3 or Google Cloud Storage, consider using
-the [S3 input source](#s3-input-source) or the [Google Cloud Storage input source](#google-cloud-storage-input-source) instead.
+You can also ingest from other storage using the HDFS input source if the HDFS client supports that storage.
+However, if you want to ingest from cloud storage, consider using the proper input sources for them.
```

Comment:
```suggestion
However, if you want to ingest from cloud storage, consider using the service-specific input source for your cloud storage.
```

Reply:
You can read from not only cloud storage but any storage if it's supported by the HDFS client. I changed to `the service-specific input source for your data storage`.